upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -179,6 +179,12 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 		dest["CiliumSecret"] = func() string { return ciliumsecretString }
 	}
 
+	if cluster.Spec.Networking != nil && cluster.Spec.Networking.Calico != nil {
```

Comment:
You don't need this one. You can just the value directly. Already doing the same for Cilium.

Reply:
Ah got it. Thanks for the tip.